src/classes/issues-processor.ts

```diff
@@ -944,6 +959,65 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUnstale(
+    issue: Issue,
+    removeLabels: Readonly<string>[]
+  ): Promise<void> {
+    if (!removeLabels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing all the labels specified via the ${this._logger.createOptionLink(
+        Option.LabelsToRemoveWhenUnstale
+      )} option.`
+    );
+
+    for (const label of removeLabels.values()) {
+      await this._removeLabel(issue, label);
+    }
+  }
+
+  private async _addLabelsWhenUnstale(
+    issue: Issue,
+    addLabels: Readonly<string>[]
```

Comment:
```suggestion
    labels: Readonly<string>[]
```

Reply:
haven't been ignoring this suggestion. Trying to keep the var's name as `labels` would result in an error passing `labels: labels` when calling the API, presumably because the names were identical.